gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributeView.java

```diff
@@ -59,19 +62,19 @@ public void setTimes(FileTime lastModifiedTime, FileTime lastAccessTime, FileTim
   public boolean equals(@Nullable Object other) {
```

Comment:
Yes, guava is one however we should avoid transitive dependency.

I guess for now we should keep explicitly depend on `com.google.code.findbugs` and maybe
change that later when the we decide that the checker option for JSR-305 is more official.
I did raise this question also to the guava people.